### ISSUE TITLE
When a user verifies their account the UI continues to believe they are unverified

### ISSUE THREAD

**Issue by addshore** [wbstack/ui#22]
Currently the flow is:
 - Create account
 - Receive user verification email
 - click link in email to verify
 - you are now "verified" on the backend, but the API still says you need to verify
 - You can then become verified in the UI by logging out (or by clicking the button to send you another email).

Instead at the point that a user is verified when they are logged in the UI should record that.
OR the user should just be logged out after verification (easier)

**Comment by AlaaSarhan** [comment]
I'll go with the first option, better UX :) and it shouldn't be that much more effort

**Comment by AlaaSarhan** [comment]
I can't seem to be able to reproduce this locally with a mocked api

**Comment by addshore** [comment]
I think I managed to reproduce this just now on 2.3 (with the patch that was just merged in it).

I believe the steps to reproduce were:
 - Create a new account
 - Leave a tab open with that new account in its unverified state
 - Click the verification link in the email (opening a second tab)

At this point the old tab will still say not verified and the new tab will say verified successfully.

If you then navigate to the account page (or probably any page) using the OLD tab that still said NOT verified then the UI state will seemingly reset totally to its unverified state.

**Comment by AlaaSarhan** [comment]
@addshore yup I suspected that was meant. So for this to work we would need to fetch again verified state in an open tab periodically, or we open a web-socket to push the update from the server.

First is less effort. But may be least effort of all is logging out all sessions on verification.

I'll go with the first I guess.

**Comment by addshore** [comment]
> First one is less effort. But may be least effort of all is logging out all sessions on verification.

Yeah that sounds pretty good to me! :)

**Comment by AlaaSarhan** [comment]
This works btw? should we close it?

**Comment by addshore** [comment]
Yes this was https://github.com/wbstack/ui/pull/25 !
